lib/oxrdf/src/literal.rs

```diff
@@ -37,7 +37,7 @@ use std::option::Option;
 pub struct Literal(LiteralContent);
```

Comment:
You might want to make `LiteralContent` `pub` here to make it accessible.

Reply:
I'm not quite sure what the effect of doing this is, I was assuming it would allow

```rust
pub fn from_term(object: Term) -> Option<String> {
    match object {
        Term::NamedNode(iri) => None,
        Term::BlankNode(id) => None,
        Term::Literal(Literal(LiteralContent::LanguageTaggedString { value, language })) => None,
        Term::Literal(Literal(LiteralContent::String(value))) => Some(value),
        Term::Literal(Literal(LiteralContent::TypedLiteral { value, datatype })) => Some(value),
        #[cfg(feature = "rdf-star")]
        Term::Triple(_) => None,
    }
}
```

but this code still results in the error

```
note: enum `crate::literal::LiteralContent` exists but is inaccessible
label: use of undeclared type `LiteralContent`
```

Reply:
You need to add a `pub use crate::literal::Literal` in the `lib.rs` file.